docs/upgrade/upgrade-to-enterprise/upgrade-guide-from-5.1-to-2022.2/metric-update-5.1-to-2022.2.rst

```diff
@@ -0,0 +1,5 @@
+=================================================================
+ScyllaDB Metric Update - Scylla 5.0 to Scylla Enterprise 2022.1
+=================================================================
+
+There are no metric updates in  ScyllaDB Enterprise 2022.1 compared to ScyllaDB 5.0.
```

Comment:
Just curious (I'll still merge this PR) - if all this file says is that it has nothing to say, why have this file at all?

Reply:
Its a place holder, in case we do find metrics updates later